src/animation/use-animate-prop.ts

```diff
@@ -59,6 +60,14 @@ export function useAnimateProp(
         prevValues.current = targetWithoutTransition(targetAndTransition, true)
     }
 
+    const currentInvalidateAnimate = useRef()
+    const prevInvalidateAnimate = useRef()
+
+    currentInvalidateAnimate.current = invalidateAnimate
+    useEffect(() => {
+        prevInvalidateAnimate.current = invalidateAnimate
+    })
```

Comment:
There are a few different ways to write this. Let me know if you'd prefer a different approach!

Reply:
I think it's possible just to initialise this 

```jsx
prevInvalidateAnimate = useRef(invalidateAnimate)
```

Check it on line 99

```jsx
const isAnimateInvalid = prevInvalidate.current !== invalidateAnimate
```

And then update it on line 110

```jsx
prevInvalidateAnimate.current = invalidateAnimate
```

Reply:
That's a better approach. I'll update it!